src/main/java/ggs/srr/domain/FtUser.java

```diff
@@ -39,5 +41,11 @@ public FtUser(Long resourceOwnerId, String email, String intraId, String image,
         this.wallet = wallet;
     }
 
-
+    @Override
+    public String toString() {
+        return "FtUser{" +
+                "level=" + level +
+                ", intraId='" + intraId + '\'' +
+                '}';
```

Comment:
```suggestion
        return String.format("FtUser{level=%f, intraId='%s'}", level, intraId);
```
이렇게 포맷 스트링을 사용해보면 어떨까요? 아니면 적어도 마지막에 `'\''` 나 `'}'` 가 쌍따옴표로 통일되면 좋겠어요!

Reply:
java에서 자동으로 만들어주는 toString인 것 같은데, string 이면 "", charcter면 '' 차이로 발생한 것 같습니다!
@joonwan 같이 확인 부탁드려요

Reply:
데이터 확인용으로 출력하기위헤 toString을 ide를 이용해 오버라이딩 해버렸는데 깔끔하지 못하게한 점 죄송합니다. 

@wonhyeongseo 원형님이 말씀하신 것 처럼 포멧을 정해 로그를 출력하는 방향이 좋다고 생각합니다!!